.github/workflows/issue-reaction-reminder.yml

```
@@ -0,0 +1,51 @@
+            const issue = context.payload.issue;
+
+            // Only run for issues labeled "car" or "powerwall"
+            const allowedLabels = ["car", "powerwall"];
+            const hasRelevantLabel = issue.labels.some(label =>
+              label.name === "car" || label.name === "powerwall"
+            );
+
+            if (!hasRelevantLabel) return;
+
+            const reminderText = `If you’re affected by this issue, unless you have valuable information to add, please use the reaction 👍  instead of commenting "same here".  
+This helps keep the thread clean and easier to track.`;
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)